docs/src/modules/components/AppLayoutDocs.js

```diff
@@ -108,14 +106,14 @@ function AppLayoutDocs(props) {
           </AdGuest>
         )}
         <Main disableToc={disableToc}>
+          {disableToc ? null : <AppTableOfContents toc={toc} />}
```

Comment:
Perhaps a comment to explain the reversed order of things? To avoid this from regressing again.